api/controllers/PublicController.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
           throw new Error('Email not recognised.')
         } else {
           randomString = ForgotPass.randomString()
-          return Members.update({ id: member.id }, { new_password: randomString })
+          return Members.update({ id: member.id }, { new_password: randomString, id: member.id })
```

Comment:
This was to fix breaking tests. The way I have implemented it `beforeUpdate` looks for the member using their `id`. If the `id` is not in the request then `beforeUpdate` does not have the information to find the member in DB.